src/data/nuclear-player-bin/.SRCINFO.template.ts

```diff
@@ -1,20 +1,19 @@
 export default `pkgbase = nuclear-player-bin
-pkgdesc = A free, multiplatform music player app that streams from multiple sources.
-pkgver = {{pkgver}}
-pkgrel = 2
-url = http://nuclear.js.org
-install = nuclear-player-bin.install
-arch = x86_64
-license = AGPL-3.0
-depends = libnotify
-depends = libappindicator-gtk3
-depends = libxtst
-depends = nss
-source = {{deburl}}
-source = nuclear.desktop
-md5sums = {{debmd5}}
-md5sums = {{desktopmd5}}
+	pkgdesc = A free, multiplatform music player app that streams from multiple sources.
+	pkgver = {{pkgver}}
+	pkgrel = 3
```

Comment:
I am not sure how your system could handle `pkgrel`. 

https://wiki.archlinux.org/title/PKGBUILD#pkgrel
https://man.archlinux.org/man/core/pacman/PKGBUILD.5.en#OPTIONS_AND_DIRECTIVES

I set it to `3` because it was set to `2` and this would be a new package release of the same version of the package